### PR TITLE
Add "Example aliases" to README

### LINKED ISSUE #301: Add "Preferred aliases" to README
```sh
alias fm='fzf-make'
alias ff='fzf-make --repeat'
alias fh='fzf-make --history'
```

### PR DESCRIPTION
**Pull request by mei28**
Resolves #301 

* Added a section for `Example aliases` in the README.
* Removed paths to non-English versions of README.md and retained only the English version.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -109,6 +103,27 @@ Whether `package.json` and `yarn.lock` are in the current directory.
...
+* For bash: `~/.bashrc`
+* For zsh: `~/.zshrc`
+* For fish: `~/.config/fish/config.fish` (use alias syntax as shown above)
+1. Reload your shell configuration:
+bash/zsh: `source ~/.bashrc` or `source ~/.zshrc`
+fish: `source ~/.config/fish/config.fish`
+### Example Usage
+* `fm`: Launch fzf-make to select and run a target.
+* `fr`: Repeat the last executed command.
+* `fh`: Open fzf-make with the history pane focused.
```
Comment (kyu08) [inline README.md:125]: I think it's enough that showing example settings only. 👍
So please delete L114-L125.

```suggestion
```

File: README.md [hunk 2]
```diff
@@ -9,12 +9,6 @@
 [![crates.io](https://img.shields.io/crates/v/fzf-make?style=flatflat-square)](https://crates.io/crates/fzf-make)
 [![Crates.io Total Downloads](https://img.shields.io/crates/d/fzf-make)](https://crates.io/crates/fzf-make)
 
-<p align="center">
-    [English]
-    [<a href="doc/README-de.md">Deutsch</a>]
-    [<a href="doc/README-fr.md">Français</a>]
-</p>
-
```
Comment (kyu08) [inline README.md:17]: I'm sorry for giving you a misunderstanding.
Please revert this deletion. 🙏

In description of https://github.com/kyu08/fzf-make/issues/301, I wanted to show that READMEs other than English just  need not to modified.

I'm going to delete them by another PR.

**Review by kyu08 — COMMENTED** [review]
Thank you for submitting and taking your time for this! 💪
Please check my comments. Please ask me if you have any question.

File: README.md (hunk 1 above)
Comment (mei28) [reply README.md]: fixed

File: README.md (hunk 2 above)
Comment (mei28) [reply README.md]: fixed

**Comment by mei28** [comment]
@kyu08 
I've fixed. please check it!

**Review by kyu08 — APPROVED** [review]
Great work!

Thank you for the contribution!

**Pull request merged**